src/main/java/com/compilercharisma/chameleonbusinessstudio/entity/AppointmentEntity.java

```diff
@@ -20,108 +20,46 @@
  * 
  * @author Matt Crow <mattcrow19@gmail.com>
  */
-@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 public class AppointmentEntity implements Serializable {
-    @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    @Setter
-    @Getter
-    @Column(name="appt_id")
-    private int id;
+    private  String _id;
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private LocalDateTime startTime;
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private LocalDateTime endTime;
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private String title;
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private String location = "online"; // default to online appointment
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private String description = "";
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private String restrictions = "";
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    private boolean isCanceled = false;
-    
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    private int totalSlots = 1; // default to one person per appointment
-    
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    @ElementCollection
-    @CollectionTable(joinColumns = @JoinColumn(name = "appt_id"))
-    @JoinColumn(name = "appt_id")
-    @OnDelete(action = OnDeleteAction.CASCADE)
-    private Set<String> tags = new HashSet<>();
-    
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    @ElementCollection
-    @CollectionTable(joinColumns = @JoinColumn(name = "appt_id"))
-    @JoinColumn(name = "appt_id")
-    @OnDelete(action = OnDeleteAction.CASCADE)
-    private Set<String> registeredUsers = new HashSet<>();
-    /*
-    registeredUsers column should have an FK relation to user email, but I don't
-    think it's possible to specify in JPA. Regardless, we'll be migrating to
-    Vendia in the near future, so it likely isn't worth investing time into.
-    */
+
+    private Boolean isCanceled = false;
     
+
+    private Integer totalSlots = 1; // default to one person per appointment
+
+    private List<String> participants;
```

Comment:
Since we're migrating from `Set` to `List`, please add a unit test to ensure duplicate participants cannot be added to an appointment, as `Set`s have built-in checking for this, whereas `List`s do not.